airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ReplicationView.tsx

```diff
@@ -143,15 +131,15 @@ export const ReplicationView: React.FC<ReplicationViewProps> = ({ onAfterSaveSch
     }
   };
 
-  const onSubmitForm = async (values: ValuesProps): Promise<void> => {
+  const onSubmitForm = async (values: ValuesProps): Promise<SubmitCancel> => {
```

Comment:
It seems unnatural to me that a `submitXYZ` function would return a promise of `SubmitCancel`.  Maybe `SubmitStatus`?

Reply:
I'm still not sure I like this whole pattern. Maybe I need to invert it.

Reply:
I think mentally `cancel: true` is less intuitive than `complete: false` or `success: false` or something like that.